### ISSUE TITLE
Issue with commit on Windows

### ISSUE THREAD

**Issue by jdfa** [extrawurst/gitui#1463]
**Describe the bug**
Trying to perform my first commit using gitui, getting such error:
```
Error
commit error:
io error:program not found
```

Line from log:
`[ERROR] hookspath error: config value 'core.hooksPath' was not found; class=Config (7); code=NotFound (-3)`

**To Reproduce**
1. Stage some changes.
2. `C` for commit command.
3. Type commit message.
4. Enter.
5. Error.

**Expected behavior**
Completed commit or more verbose error message.

**Screenshots**
![image](https://user-images.githubusercontent.com/14260814/205919720-8b74b3c0-c650-42d7-b4a3-08ae2beb4e84.png)

**Context (please complete the following information):**
 - Windows 10
 - GitUI 0.22.1

**Comment by jdfa** [comment]
Yes, I do. It is pre-commit hook which validates that I am not committing TODO comments.
But it should pass that specific commit.

**Comment by extrawurst** [comment]
Can you share the path of the hook and the content of the script? also tell me if you use `core.hooksPath` and if so how it is configured

**Comment by jdfa** [comment]
I am not using `hooksPath` setting. Hook itself is located inside .git folder, path: repoRoot/.git/hooks/pre-commit.
Hook itself is redirecting to powershell script, located in common folder:
```
#!/bin/sh

echo
exec powershell.exe -NoProfile -ExecutionPolicy Bypass -File 'D:\ps\hook.ps1'
exit

```

Just to be complete, here is content of hook.ps1:
```
﻿if((git diff --cached) -like "*TODO*"){
    Write-Output "TODO is present in pending changes, commit rejected"
    exit 1
}

```

It works fine with git command, also handled properly with VisualStudio Git integration.

**Comment by pm100** [comment]
@extrawurst I repro'd this using 0.22.1 but its fine in master head. I think it was fixed by https://github.com/extrawurst/gitui/pull/1532,

**Comment by extrawurst** [comment]
Let’s see after next release then

**Comment by pm100** [comment]
@extrawurst - ignore that last message, bug seems to be back (I'm probably trying to do too many thigns at once!)
